src/System.Management.Automation/engine/lang/parserutils.cs

```diff
@@ -1036,6 +1036,13 @@ private static object ReplaceOperatorImpl(ExecutionContext context, string input
         /// <returns>The result of the operator.</returns>
```

Comment:
@KirkMunro, your last commit had 4 failures in `PowerShell-CI-macos`
Read-Host.[Console]::ReadKey() implementation on non-Windows.Should output correctly
```powershell
Expected 0, but got 1.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Host/Read-Host.Tests.ps1: line 25
25:             $LASTEXITCODE | Should -Be 0
```
Invoke-WebRequest tests.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:
The term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
+ CategoryInfo          : ObjectNotFound: (WebListener:String) [Get-Command], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
+ PSComputerName        : localhost
 
Cannot bind argument to parameter 'Path' because it is null.
+ CategoryInfo          : InvalidData: (:) [Split-Path], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.SplitPathCommand
+ PSComputerName        : localhost
 
The expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : BadExpression
+ PSComputerName        : localhost
 

Output:
appEXE: 
serverPfxPath: /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/ServerCert.pfx
serverPfxPassword: 47461eaa96
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /Users/vsts/agent/2.158.0/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```
Invoke-RestMethod tests.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:
The term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
+ CategoryInfo          : ObjectNotFound: (WebListener:String) [Get-Command], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
+ PSComputerName        : localhost
 
Cannot bind argument to parameter 'Path' because it is null.
+ CategoryInfo          : InvalidData: (:) [Split-Path], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.SplitPathCommand
+ PSComputerName        : localhost
 
The expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : BadExpression
+ PSComputerName        : localhost
 

Output:
appEXE: 
serverPfxPath: /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/ServerCert.pfx
serverPfxPassword: b0b62430cc
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /Users/vsts/agent/2.158.0/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```
Validate Invoke-WebRequest and Invoke-RestMethod -InFile.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:
The term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
+ CategoryInfo          : ObjectNotFound: (WebListener:String) [Get-Command], CommandNotFoundException
+ FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
+ PSComputerName        : localhost
 
Cannot bind argument to parameter 'Path' because it is null.
+ CategoryInfo          : InvalidData: (:) [Split-Path], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.SplitPathCommand
+ PSComputerName        : localhost
 
The expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : BadExpression
+ PSComputerName        : localhost
 

Output:
appEXE: 
serverPfxPath: /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/ServerCert.pfx
serverPfxPassword: 9bd30b2d4e
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /Users/vsts/agent/2.158.0/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```